src/AllBuiltNodes.hpp

```diff
@@ -34,7 +44,7 @@ struct BuiltNode
 
 struct AllBuiltNodes
 {
-    static const uint32_t MagicNumber = 0x53532dc2 ^ kTundraHashMagic;
+    static const uint32_t MagicNumber = 0x53533dc2 ^ kTundraHashMagic;
```

Comment:
Please document why we need a MagicNumber here and perhaps name it with a proper variable name

Reply:
The structs in tundra with magic numbers are "frozen" to disk, then mmap'ed next time tundra is run. The magic number is changed whenever the struct is changed, so tundra can detect if it's trying to read a frozen data file with the format it is expecting or if it's from a different tundra version (in which case the mmap'ed data cannot be trusted/used).